README.md

```diff
@@ -113,6 +113,196 @@ The below Environment variables are optional. If not provided a manual Applicati
+      </td>
+    </tr>
+    <tr>
+      <td>
+       BOLD_SERVICES_SITE_IDENTIFIER
+      </td>
+      <td>     
+       This is site identifier, and it will be the part of the application URL.
+      <br />
+      If the value is not given, the site will be deployed using the default value.
+      </td>
+    </tr>
+</table>
+<br/>
+
+
+## Auto deployment docker run command.
+
+### Without branding
+
+```sh
+docker run --name boldbi -p 80:80 -p 443:443 \
+      -e APP_URL=<app_base_url> \
+      -e OPTIONAL_LIBS=<optional_library_names> \
+      -e widget_bing_map_enable=<true/false>\
+      -e widget_bing_map_api_key=<widget_bing_map_api_key> \
+      -e BOLD_SERVICES_UNLOCK_KEY=<Bold_BI_license_key>  \
+      -e BOLD_SERVICES_DB_TYPE=<data_base_server_type>  \
+      -e BOLD_SERVICES_DB_HOST=<data_base_server_host> \
+      -e BOLD_SERVICES_DB_PORT=<data_base_server_port> \
+      -e BOLD_SERVICES_DB_USER=<data_base_user_name> \
+      -e BOLD_SERVICES_DB_PASSWORD=<data_base_server_password> \ 
+      -e BOLD_SERVICES_DB_NAME=<excisting_data_base_name> \
+      -e BOLD_SERVICES_POSTGRESQL_MAINTENANCE_DB=<maintenance_db_name> \
+      -e BOLD_SERVICES_USER_EMAIL=<Bold_BI_user_email> \
+      -e BOLD_SERVICES_USER_PASSWORD=<Bold_BI_user_password> \ 
+      -v <host_path_for_appdata_files>:/application/app_data \
+      -v <host_path_for_nginx_config>:/etc/nginx/sites-available \
+      -d syncfusion/boldbi:<tag>
+```
+
+### With branding
+
+```sh
+docker run --name boldbi -p 80:80 -p 443:443 \
+      -e APP_URL=<app_base_url> \
+      -e OPTIONAL_LIBS=<optional_library_names> \
+      -e widget_bing_map_enable=<true/false>\
+      -e widget_bing_map_api_key=<widget_bing_map_api_key> \
+      -e BOLD_SERVICES_UNLOCK_KEY=<Bold_BI_license_key>  \
+      -e BOLD_SERVICES_DB_TYPE=<data_base_server_type>  \
+      -e BOLD_SERVICES_DB_HOST=<data_base_server_host> \
+      -e BOLD_SERVICES_DB_PORT=<data_base_server_port> \
+      -e BOLD_SERVICES_DB_USER=<data_base_user_name> \
+      -e BOLD_SERVICES_DB_PASSWORD=<data_base_server_password> \ 
+      -e BOLD_SERVICES_DB_NAME=<excisting_data_base_name> \
+      -e BOLD_SERVICES_POSTGRESQL_MAINTENANCE_DB=<maintenance_db_name> \
+      -e BOLD_SERVICES_USER_EMAIL=<Bold_BI_user_email> \
+      -e BOLD_SERVICES_USER_PASSWORD=<Bold_BI_user_password> \ 
+      -e BOLD_SERVICES_BRANDING_MAIN_LOGO=<branding_image_url> \
+      -e BOLD_SERVICES_BRANDING_LOGIN_LOGO=<branding_image_url> \
+      -e BOLD_SERVICES_BRANDING_EMAIL_LOGO=<branding_image_url> \
+      -e BOLD_SERVICES_BRANDING_FAVICON=<branding_image_url> \
+      -e BOLD_SERVICES_BRANDING_FOOTER_LOGO=<branding_image_url> \
+      -e BOLD_SERVICES_SITE_NAME=<site_name> \
+      -e BOLD_SERVICES_SITE_IDENTIFIER=<site_identifier_name> \
+      -v <host_path_for_appdata_files>:/application/app_data \
+      -v <host_path_for_nginx_config>:/etc/nginx/sites-available \
+      -d syncfusion/boldbi:<tag>
+```
+## Example for Auto deployment docker run command.
```

Comment:
Example for auto deployment docker run command.

Reply:
concern addressed